File: csharp/MapleStory.OpenAPI/Src/KMS/Dto/Guild/Guild.cs
using Newtonsoft.Json;
using Base = MapleStory.OpenAPI.Common.DTO;

namespace MapleStory.OpenAPI.KMS.DTO
{
    /// <summary>
    /// 길드 식별자 정보
    /// </summary>
    public class GuildDTO : Base.GuildDTO
    {
        /// <summary>
        /// 길드 식별자
        /// </summary>
        [JsonProperty("oguild_id")]
        public override string OGuildId { get; set; }
    }
}

File: js/src/maplestory/api/tms/dto/guild/guild.ts
import * as base from '../../../common/dto/guild/guild';
import { GuildBody } from '../../response/guild/guildBody';

/**
 * 公會識別碼資訊
 */
export class GuildDto extends base.GuildDto {
  /**
   * 公會識別碼
   */
  public override oguildId: string;

  constructor(obj: GuildBody) {
    super();

    const { oguild_id } = obj;

    this.oguildId = oguild_id;
  }
}
